CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+### 1.10.1
+
+CHANGES:
+* Update NGINX version to 1.19.8.
+* Add Kubernetes 1.20 support.
+* Fix various issues in the Makefile.
```

Comment:
would it make sense to mention the main problem here. For example,

* [1373](https://github.com/nginxinc/kubernetes-ingress/pull/1373): Fix various issues in the Makefile. In 1.10.0, a bug was introduced that prevented building Ingress Controller images on versions of `make` < X.Y.Z.

(I also added a link to the PR)

Reply:
I wanted to link the PRs (3) for that change but noticed that we don't usually do it for minor releases. 👍